src/app/components/source/UserPrivacy.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { compose } from 'redux';
```

Comment:
Hello @deepakchethan! Sorry, I got a bit busy with some other stuff! Thanks for pinging!
Is this really necessary? We're trying to get rid of redux

Reply:
Are you still around @deepakchethan?

Reply:
Hey @amoedoamorim, Sorry was not around. Will update the code

Reply:
Have removed it